packages/client/hmi-client/src/components/resources/resources-list.vue

```diff
@@ -0,0 +1,90 @@
+<script setup lang="ts">
+import { Project, ProjectAssetTypes } from '@/types/Project';
+import { RouteMetadata, RouteName } from '@/router/routes';
+import { Resource } from '@/types/Resource';
+import ResourceCard from '@/components/resources/ResourceCard.vue';
+import useResourcesStore from '@/stores/resources';
+import { RouteParamsRaw, useRouter } from 'vue-router';
+
+const props = defineProps<{
+	project: Project;
+	resourceRoute?: RouteName;
+	// maybe add list size later
+}>();
+
+const emit = defineEmits(['show-data-explorer']);
+const goToDataExplorer = () => emit('show-data-explorer');
+
+const router = useRouter();
+const store = useResourcesStore();
+
+const resources: Resource[] = [];
+
+const filteredRouteMetadata: {
+	route: RouteName;
+	metadata: { displayName: string; icon: any; projectAsset?: ProjectAssetTypes };
+}[] = props.resourceRoute
+	? [{ route: props.resourceRoute, metadata: RouteMetadata[props.resourceRoute] }]
+	: Object.entries(RouteMetadata)
+			.map(([route, metadata]) => [{ route, metadata }])
+			.flat();
+
+for (let i = 0; i < filteredRouteMetadata.length; i++) {
```

Comment:
Suggestion: convert these `for` loops to  `.forEach()` to make it a bit more terse.